src/configure/configurers/provisioningConfigurer.ts

```diff
@@ -106,6 +106,8 @@ export class ProvisioningConfigurer implements IProvisioningConfigurer {
             Messages.discardPipeline);
         let provisioningServiceResponse: ProvisioningConfiguration;
         if (!!commitOrDiscard && commitOrDiscard.toLowerCase() === Messages.commitAndPush.toLowerCase()) {
+            telemetryHelper.setCurrentStep('CreateACRResourceGroupAndSPN');
```

Comment:
Change the current step name, it should be generic name.

Reply:
changed to ConfiguringPreRequisiteParamsForCompleteMode